kahuna/public/js/usage-rights/usage-rights-editor.js

```diff
@@ -152,7 +152,8 @@ usageRightsEditor.controller(
             const image = usageRights.image;
             const resource = image.data.userMetadata.data.usageRights;
             return editsService.update(resource, data, image).
-                then(resource => resource.data);
+                then(resource => resource.data).
+                then(() => setMetadataFromUsageRights(image));
```

Comment:
I meant the value the Promise is resolved with. The returned value is now different, but it seems like it's ignored, so should be OK?